### ISSUE TITLE
An option to turn on notifications for _each_ message in conferences

### ISSUE THREAD

**Issue by EgorBo** [ForNeVeR/kaiwa#62]
Currently with "enable alerts" turned on it sends them only if somebody mentions your nick, e.g.:
EgorBo: hey!
But it'd be nice to have an option to turn on notifications for ALL messages in the conference (and it also would be nice to have that option in the chat view).

**Comment by vitalyster** [comment]
Still relevant? I hear one (ugly) sound for each message and another one for mentions. Or this ticket is about browser notifications?

**Comment by EgorBo** [comment]
@vitalyster yeah, I do hear sounds, but I would also like to have notifications for each msg in order to be able to read (preview) them without opening the Kaiwa tab :)
